Main/lib/xml/workflow/interpro.xml

```diff
@@ -1,76 +1,58 @@
 <workflowGraph name="interproWorkflow">
-
+    <param name="projectName"/>
+    <param name="organismAbbrev"/>
+    <param name="genomeExtDbRlsSpec"/>
     <param name="proteinsFile"/>
     <param name="parentDataDir"/>
     <param name="insertVersion" default="1"/>                    <!-- optional -->
     <param name="aaSeqTable"    default="ExternalAASequence"/>   <!-- optional, used if version > 1 -->
     <param name="sourceIdRegex" default="(\S+)"/>                <!-- optional, used if version > 1 -->
+    <param name="ncbiTaxonId"/>
+    <param name="gusConfigFile"/>
 
     <constant name="dataDir">$$parentDataDir$$/interpro</constant>
+    <constant name="resultsDirectory">$$dataDir$$/nextflowBaseDir/results</constant>
 
     <step name="makeDataDir" stepClass="ReFlow::StepClasses::MakeDataDir">
       <paramValue name="dataDir">$$dataDir$$</paramValue>
     </step>
 
-    <step name="makeResultDir" stepClass="ReFlow::StepClasses::MakeDataDir">
-      <paramValue name="dataDir">$$dataDir$$/results/</paramValue>
-      <depends name="makeDataDir"/>
-    </step>
-
     <step name="makeInterproNextflowConfig" stepClass="ApiCommonWorkflow::Main::WorkflowSteps::MakeIprscan5NextflowConfig">
-      <paramValue name="outputDir">$$dataDir$$/results/</paramValue>
+      <paramValue name="outputDir">$$resultsDirectory$$/</paramValue>
       <paramValue name="configFileName">nextflow.config</paramValue>    
       <paramValue name="analysisDir">$$dataDir$$</paramValue>
       <paramValue name="input">$$proteinsFile$$.NoAsterisks</paramValue>
       <paramValue name="fastaSubsetSize">100</paramValue>
       <paramValue name="appls">pfama</paramValue>
       <paramValue name="outputFile">iprscan_out.tsv</paramValue>
-      <paramValue name="interproscanDatabase">/home/rdemko/interproData/data</paramValue>
```

Comment:
this should not be specified in the xml.  Add as a stepsShared.prop (it is needed by BOTH MakeIprscan5NextflowConfig.pm and NextflowResultsCache.pm)